src/app/app.component.html

```diff
@@ -15,6 +15,15 @@
         &nbsp;&nbsp;Smart Tutor
       </span>
       <span fxFlex></span>
+      <span *ngIf="trainee">
+        <button mat-button [matMenuTriggerFor]="menu">{{trainee.studentIndex}}<mat-icon>expand_more</mat-icon></button>
+        <mat-menu #menu>
+          <button mat-menu-item (click)="onLogout()">Log out</button>
```

Comment:
Not something we need to address at this point since we are not sure it will become an issue, but still worth mentioning - the responsibility of the app component is to tie the main layout together.
Currently, another responsibility is emerging related to the toolbar, which is a sign that we should be thinking about creating a header component to encapsulate the login/logout coordination.

Still, I wouldn't perform any refactoring yet, but we should keep an eye on this and extract a component when the next piece of logic populates the app component.